generator/views/funnel_analysis_view.py

```diff
@@ -0,0 +1,238 @@
+"""Class to describe a Funnel Analysis View.
```

Comment:
This is great. I think it might be worth adding one or two sentences on how this view relates to the underlying dataset (with a link): https://docs.telemetry.mozilla.org/datasets/non_desktop/events_daily/reference.html